src/app/shared/components/table/table.component.ts

```diff
@@ -71,6 +71,10 @@ export class TableComponent<T> implements OnChanges {
     return header.sortDirection === SortDirection.ASC ? Icon.UPWARD_TRIANGLE : Icon.DOWNWARD_TRIANGLE
   }
 
+  public getSortIconTopPostion(header: SofieTableHeader<T>): number {
+    return header.sortDirection === SortDirection.ASC ? 3 : -3
```

Comment:
`3` and `-3` are magic numbers. Would be nice if they atleast got a variable with a name.

Reply:
Done